stout_a07b.js

```diff
@@ -0,0 +1,101 @@
+/*Kyle Stout
+  stout_a02b
+  INFO 2124WW
+  Thoendel
+  02-02-2020
+*/
+
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+const standardInput = process.stdin;            
+standardInput.resume();
+standardInput.setEncoding('utf8');
+const badFoods = [];
+const groceryItems = [];
+clearScreen();
+console.log(getPrompt(groceryItems));
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+
+//Step 1
+buildFoodsList (badFoods, "bread", "beer", "wine", "vodka", "gin");        
+
+standardInput.on('data', function (text) {
+    //This line will remove line breaks 
+    //\n\r in window, \r in macOS
+    text = text.replace('\n', '').replace('\r', '');
+    //Step 4 Begin
+    if (text == 'Q' || text == 'q') {
+        clearScreen();
+        console.log("Bye");
+        process.exit();
+    } else if (text == 'V' || text == 'v'){
+        clearScreen();
+        viewList();
+        getPrompt();
+    }
+    else {
+        clearScreen();
+        checkitem(text);
+        if (Boolean === false) {
+            console.log("Item added to list!");
+            groceryItems.push(text);
+            getPrompt();
+        } else {
+            console.log(`   Error: That item is not safe for your allergies.
+            It has not been added to your list.`);
+            getPrompt();
+        }
+    }
+    
+    //Step 4 End
+});
+
+/* DEFINE YOUR FUNCTIONS BETWEEN THIS LINE */
+
+//Step 2
+function buildFoodsList (badFoods, food1, food2, food3, food4, food5) {

```

Comment:
here you need to accept a variable number of arguments....
so:
buildFoodsList(badFoods, ...foods)